android/app/build.gradle

```diff
@@ -144,13 +144,9 @@ android {
     }
     buildTypes {
         release {
-            applicationIdSuffix "release"
```

Comment:
Removed this so that the v3 is being installed on top of v2 and not a separated app on android